tools/Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN cd /tmp \
   && mkdir -p /tmp/build/.modcache \
   && cd /tmp/tools \
   && go install -trimpath -tags=tools github.com/golangci/golangci-lint/cmd/golangci-lint \
-  && chmod -R 777 /tmp/build/
+  && chmod -R 777 /tmp/build/ \
+  && git config --global --add safe.directory /build
```

Comment:
We found the actual problem for the linter error (a Golang bug) and a fix that allows us to determine more easily if it's fixed in a future version. See https://github.com/lightninglabs/pool/blob/master/tools/Dockerfile#L6.

Reply:
Thanks, sound much better.